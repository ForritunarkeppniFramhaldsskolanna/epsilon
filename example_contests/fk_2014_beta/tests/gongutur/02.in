5
1 4
2.1 5
3 4
2.5 4.2
1.2 3.8
