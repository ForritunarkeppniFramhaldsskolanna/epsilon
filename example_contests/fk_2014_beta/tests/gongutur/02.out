4.73063281102622
